L1Trigger/L1THGCal/python/customCalibration.py

```diff
@@ -19,3 +19,10 @@ def custom_cluster_calibration_layers(process,
     parameters_c2d.layerWeights = weights
     parameters_c2d.applyLayerCalibration = cms.bool(True)
     return process
+
+
+def custom_fe_thresholds(process, fe_threshold=1.5, cl3d_seed_threshold=7.5):
```

Comment:
It shouldn't be needed to define a new custom function since there are already customs for frontend selections and for clustering which could be used.

Reply:
I wanted to keep it minimal, indeed I can redefine the FE and clustering configuration but I am not sure it makes the maintenance easier...

Reply:
I find it a bit strange to have this function that modifies two very different things: frontend TC thresholds and cluster seeding threshold.

An other way, which doesn't redefine the entire configs, could be to do something like that:
https://github.com/cms-sw/cmssw/blob/master/SimCalorimetry/HGCalSimProducers/python/hgcalDigitizer_cfi.py#L298-L300

Reply:
Well they are modified in the same function because the reason is the same: adapt to the new calibration. 

I probably prefer your second suggestion:
   * pros: the changes are coded where the configuration is defined ensuring consisntency
   * cons: not having all the changes in a single place: we will need to remember that the various bits and pieces are scattered around in the code.

BTW, while we discuss this: is there any reason why you use the syntax:
`parameters_c2d.applyLayerCalibration = cms.bool(True)`
instead of
`parameters_c2d.applyLayerCalibration = True`

The second has the advantage that if the configuration changes and the parameter disappears you immediately realize while the first one will simply define a new parameter, isn't it?

Reply:
If the goal of the function is to be used only to rescale all parameters in mip units by the same scale factor, then it might be even better to have a custom function but with a single parameter (the scale factor). This way we are sure all the mip parameters are rescaled consistently, the goal of the function is clearer, and the scale factor can be eventually derived from reco parameters.

Concerning your question about `cms.bool()`, I didn't really think about this; so there is no fundamental reason it is written like that.